### ISSUE TITLE
Building cmake image hangs

### ISSUE THREAD

**Issue by fkorotkov** [cirruslabs/docker-images-windows#12]
Building image via [cmake Dockerfile](https://github.com/cirruslabs/docker-images-windows/blob/master/contrib/cmake/Dockerfile) just hangs on the latest `windows-server-2019-dc-core-for-containers-v20200813` image VM on GCP.

It seems the `docker build` finishes all steps and just hangs. See this output https://cirrus-ci.com/task/5989635299999744?command=build_cmake#L530

**Comment by ferventcoder** [comment]
Any idea how long that refreshenv bit takes? I see it mentions timeout, but following the script, refreshenv is the last thing that is called, which completes. I'm guessing it completes and then just sits there until it times out.

Might be a good idea to move refreshenv off to its own line or take it out if you are not going to do any further commands afterwards. When you next start a session with the docker image, everything should be refreshed with the new shell.

**Comment by fkorotkov** [comment]
Yeah, I added the `refreshenv` to make sure `choco insall` completes but I have no idea what happens after. It does seem like Docker just hangs preparing the layer or something else. I wonder how the choco install could cause such behaviour

**Comment by ferventcoder** [comment]
I'd add one additional command after, something simple like a ping. See if it times out after that.

And still remove the refreshenv. 😀

**Comment by ericoporto** [comment]
Can we have an updated version with CMake?
